### PR TITLE
feat: multilanguage support

### PR DESCRIPTION
**Pull request by maurolucc**
Provide multilanguage support for identity provider login page.

### CODE REVIEW

**Review by xvilapueyo — APPROVED** [review]

**Review by AlexD10S — APPROVED** [review]

File: packages/identity-consent/views/index.hbs [hunk 1]
```diff
@@ -10,30 +10,42 @@
...
+  <div id='login-app'>
     <div class="leftSide">
       <img id="logo" class="logo" src="/static/logoHeader.png"></img>
+      {{!-- TODO: uncomment or delete if choosing lang is it not necessary --}}
+      {{!-- <div class="locale-changer">
+        <select v-model="$i18n.locale">
+          <option v-for="(lang, i) in langs" :key="`Lang${i}`" :value="lang" v-text="lang"></option>
+        </select>
+      </div> --}}
       <p class="poweredByInMobile">Powered by Validated ID</p>
```
Comment (AlexD10S) [inline packages/identity-consent/views/index.hbs:36]: Needs transalation too

File: packages/identity-consent/views/index.hbs [hunk 2]
```diff
@@ -140,10 +180,10 @@
         },
         replaceName(nameScope) {
           let nameModified = nameScope;
-          if(nameScope === "openid"){
-            return  "Your DID";
+          if (nameScope === "openid") {
+            return "Your DID";
```
Comment (AlexD10S) [inline packages/identity-consent/views/index.hbs:184]: It can be transalted too

File: packages/identity-consent/views/index.hbs (hunk 1 above)
Comment (kalop) [reply packages/identity-consent/views/index.hbs:36]: done

File: packages/identity-consent/views/index.hbs (hunk 2 above)
Comment (kalop) [reply packages/identity-consent/views/index.hbs:184]: done

**Pull request merged**